test/jit/test_pdt.py

```diff
@@ -42,6 +42,12 @@ def test_str(a):
 def test_list_and_tuple(a):
     return sum(a)
 
+def test_dict(a):
```

Comment:
Does current implementation work for `nn.Module` methods and normal class methods? If so, could we add some test cases?

Reply:
Tests for `nn.Module` methods are still pending. I'm working on it